src/urllib3/connection.py

```diff
@@ -377,6 +379,9 @@ def __init__(
         self.key_password = key_password
         self.ssl_context = ssl_context
         self.server_hostname = server_hostname
+        self.ssl_version = None
```

Comment:
`self.ssl_version` isn't defined anywhere else and I thought it was strange that we were assigning to an attribute not defined in `HTTPSConnection.__init__` within `HTTPSConnectionPool`. Obviously this works fine "because Python" but figured it'd be nicer to be explicit that `ssl_version` is indeed a valid attribute of `HTTPSConnection`.